static/app/views/replays/detail/domMutations/domMutationRow.tsx

```diff
@@ -0,0 +1,197 @@
+import {CSSProperties, useCallback} from 'react';
+import styled from '@emotion/styled';
+
+import BreadcrumbIcon from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/type/icon';
+import HTMLCode from 'sentry/components/htmlCode';
+import {getDetails} from 'sentry/components/replays/breadcrumbs/utils';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+import {relativeTimeInMs} from 'sentry/components/replays/utils';
+import {SVGIconProps} from 'sentry/icons/svgIcon';
+import space from 'sentry/styles/space';
+import {getPrevReplayEvent} from 'sentry/utils/replays/getReplayEvent';
+import useCrumbHandlers from 'sentry/utils/replays/hooks/useCrumbHandlers';
+import type {Extraction} from 'sentry/utils/replays/hooks/useExtractedCrumbHtml';
+import TimestampButton from 'sentry/views/replays/detail/timestampButton';
+
+type Props = {
+  mutation: Extraction;
+  mutations: Extraction[];
+  startTimestampMs: number;
+  style: CSSProperties;
```

Comment:
What uses this prop? is it the virtualization?

Reply:
yup! the virtual table sets absolute positioning on the list-items or table-cells.